tests/tests/cpld_bmark_p7

```diff
@@ -131,6 +131,19 @@ export MOM6_RESTART_SETTING=r
 export USE_CICE_ALB=.false.
 export MIN_SEAICE=1.0e-11
 export IOPT_SFC=1
+export DNATS=1
+export IMP_PHYSICS=11
+export CCPP_SUITE=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1
+export FIELD_TABLE=field_table_gfsv16
+export LGFDLMPRAD=.true.
+export KNOB_UGWP_VERSION=1
+export DO_UGWP_V0=.false.
+export DO_UGWP_V1=.true.
```

Comment:
So we are not running ugwpv1 in p8 test? Is there a plan to add a ugwpv1 test in p8 configuration?

Reply:
This test is maintained which tests DO_UGWP_V1=true.  The p8 tests are all updated to the full set of p8b settings.  I do not know of any additional planned tests.

Reply:
Since now the P8 tests are available, do we still need to maintain the cpld_bmark_p7 test? Can we have P8+ugwdv1 test?

Reply:
I have maintained this test so it would get the same results as before.  Any P8+ugwdv1 test seems out of the scope of the current PR.